scenarios/smart_trailer/digital_twin_providers/common/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# SPDX-License-Identifier: MIT
+
+[package]
+name = "digital_twin_providers_common"
+version = "0.1.0"
+edition = "2021"
+license = "MIT"
+
+[dependencies]
+invehicle-stack-interfaces = { path = "../../proto_build" }
```

Comment:
consider putting this in the workspace and referring to it with `workspace = true`